doc/docusaurus.config.ts

```diff
@@ -18,7 +18,8 @@ const config: Config = {
   organizationName: 'polymerdao', // Usually your GitHub org/user name.
   projectName: 'monomer', // Usually your repo name.
 
-  onBrokenLinks: 'throw',
+  // TODO: Change onBrokenLinks to "throw"
+  onBrokenLinks: 'warn',
```

Comment:
Need to change back?

Reply:
I don't know the details because I didn't design this site, but there are some broken links:
```
[cause]: Error: Docusaurus found broken links!
  
  Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
  Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.
  
  It looks like some of the broken links we found appear in many pages of your site.
  Maybe those broken links appear on all pages through your site layout?
  We recommend that you check your theme configuration for such links (particularly, theme navbar and footer).
  Frequent broken links are linking to:
  - /blog
  
  Exhaustive list of all broken links found:
  - Broken link on source page path = /404.html:
     -> linking to /blog
  - Broken link on source page path = /markdown-page:
     -> linking to /blog
  - Broken link on source page path = /docs/category/learn:
     -> linking to /blog
  - Broken link on source page path = /docs/intro:
     -> linking to /blog
  - Broken link on source page path = /docs/learn/create-an-app-with-monomer:
     -> linking to /blog
  - Broken link on source page path = /docs/learn/the-rollup-module:
     -> linking to /blog
  - Broken link on source page path = /:
     -> linking to /blog
 ```
 To make the site build, we need to change error to warning.

Reply:
Ah, then this is a perfect example of something the CI can help us catch. Do you mind sending a separate PR to fix the broken links? Then we can merge this without changing `onBrokenLinks`

Reply:
Should be fixed in #137